examples/wget/Kconfig

```diff
@@ -12,11 +12,18 @@ config EXAMPLES_WGET
 
 if EXAMPLES_WGET
 
+config EXAMPLES_WGET_STACKSIZE
+	int "wget stack size"
+	default 2048
+	---help---
+		Stack size of "wget" command. If you use SSL connection,
+		you should increase this value around 8096.
+
 config EXAMPLES_WGET_URL
 	string "File URL"
 	default ""
 	---help---
-	The URL of the file to get
+		The URL of the file to get
 
 config EXAMPLES_WGET_NOMAC
```

Comment:
Should we guard this by NSH_NETINIT?

Reply:
I have no idea.
It looks that not related with NSH_NETINIT, isn't it?

Reply:
I mean change to:
```
config EXAMPLES_WGET_NOMAC
depends on !NSH_NETINIT
```
It doesn't make sense to let user select mac related option if NSH_NETINIT turn on because your code skip all related code in this case.

Reply:
I see. I've fixed it and main code.